src/app.ts

```diff
@@ -159,6 +159,11 @@ export class App implements IAppView {
    * @param isBusy
    */
   setBusy(isBusy) {
+    this.$node.select('.leftMetaBox').classed('invisibleClass', isBusy);
```

Comment:
As it would implement a smooth transition if needed. So if the elements should fade in and out. The class lets the elements fade.